.pre-commit-config.yaml

```
@@ -3,10 +3,17 @@ repos:
     rev: 'v14.0.0'
     hooks:
       - id: clang-format
+        files: ".*\\.(cpp|hpp|ipp,cu,cuh)$"
```

```suggestion
        files: ".*\.(cpp|hpp|ipp|cu|cuh)$"
```
Does this work? And if yes, why? With regex I would expect above.

---

Ah, no, my mistake!